<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>692</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>TranslationProcessor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>../icons/trProcess.png</normalon>
   </iconset>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="centralLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <property name="horizontalSpacing">
     <number>5</number>
    </property>
    <property name="verticalSpacing">
     <number>3</number>
    </property>
    <item row="1" column="0">
     <widget class="QWidget" name="translationsPanel" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="translationsLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="translationsTabs">
         <property name="tabPosition">
          <enum>QTabWidget::South</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="translationsTableTab">
          <attribute name="title">
           <string>Таблица</string>
          </attribute>
          <layout class="QHBoxLayout" name="translationsTableLayout">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>5</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QTableWidget" name="translationsTable">
             <property name="focusPolicy">
              <enum>Qt::WheelFocus</enum>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="gridStyle">
              <enum>Qt::SolidLine</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="translationsSourceTab">
          <attribute name="title">
           <string>Исходник</string>
          </attribute>
          <layout class="QHBoxLayout" name="translationsSourceLayout">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>5</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QTextEdit" name="translationsSourceTextEdit"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QWidget" name="gitButtonsPanel" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="toolTipDuration">
       <number>-2</number>
      </property>
      <layout class="QHBoxLayout" name="gitButtonsLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="gitButtonsSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="gitCommitButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>50</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Сохранить в деве (git commit)</string>
         </property>
         <property name="statusTip">
          <string>git commit</string>
         </property>
         <property name="text">
          <string>Сохранить</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/Save.png</normaloff>../icons/Save.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="gitPullButton">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>50</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Получить из сервера (Git pull)</string>
         </property>
         <property name="statusTip">
          <string>git pull</string>
         </property>
         <property name="text">
          <string>Получить</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/Download.png</normaloff>../icons/Download.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="gitPushButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>50</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Отправить на сервер (git push)</string>
         </property>
         <property name="statusTip">
          <string>git push</string>
         </property>
         <property name="text">
          <string>Отправить</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/Upload.png</normaloff>../icons/Upload.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>gitPushButton</zorder>
      <zorder>gitButtonsSpacer</zorder>
      <zorder>gitPullButton</zorder>
      <zorder>gitCommitButton</zorder>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="progressBarPanel" native="true">
      <layout class="QHBoxLayout" name="progressBarLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>925</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="settingsAction"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <widget class="QMenu" name="menuTranslations">
    <property name="title">
     <string>Переводы</string>
    </property>
    <addaction name="gitCommitAction"/>
    <addaction name="separator"/>
    <addaction name="gitPullAction"/>
    <addaction name="gitPushAction"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Вид</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuTranslations"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="projectsDock">
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>105</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Проекты</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="projectsDockContents">
    <layout class="QHBoxLayout" name="projectsDockLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QListView" name="projectsListView">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="gitBranchesDock">
   <property name="minimumSize">
    <size>
     <width>183</width>
     <height>210</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Ветки</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="gitBranchesDockContent">
    <layout class="QVBoxLayout" name="gitBranchesDockLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeWidget" name="gitBranchesTree">
       <column>
        <property name="text">
         <string>Ветка</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>WWWINT</string>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="flags">
         <set>ItemIsEnabled</set>
        </property>
        <item>
         <property name="text">
          <string>WWWINT-75456</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>WWWINT-55668</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>WWWINT-56565</string>
         </property>
        </item>
       </item>
       <item>
        <property name="text">
         <string>WWWSV</string>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="flags">
         <set>ItemIsEnabled</set>
        </property>
        <item>
         <property name="text">
          <string>WWWSV-12363</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>WWWSV-56356</string>
         </property>
        </item>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="gitBranchesButtonsContainer" native="true">
       <layout class="QHBoxLayout" name="gitBranchesButtonsLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <spacer name="gitBranchesButtonsSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="gitBranchesCreateButton">
          <property name="text">
           <string>Создать</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../icons/Add.png</normaloff>../icons/Add.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="gitBranchesUpdateButton">
          <property name="text">
           <string>Обновить</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../icons/Available Updates.png</normaloff>../icons/Available Updates.png</iconset>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="domainsDock">
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>105</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Домены</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="domainsDockContent">
    <layout class="QHBoxLayout" name="domainsDockLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QListView" name="domainsListView">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="appLogsDock">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Лог приложения</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="appLogsDockContents">
    <layout class="QHBoxLayout" name="appLogsDockLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTextBrowser" name="appLogsTextBrowser">
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot; bgcolor=&quot;#000000&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Loading...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextBrowserInteraction</set>
       </property>
       <property name="placeholderText">
        <string>Загрузка...</string>
       </property>
       <property name="searchPaths">
        <stringlist/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="quitAction">
   <property name="text">
    <string>Выйти</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="gitPullAction">
   <property name="icon">
    <iconset>
     <normaloff>../icons/Download.png</normaloff>../icons/Download.png</iconset>
   </property>
   <property name="text">
    <string>Получить</string>
   </property>
   <property name="statusTip">
    <string>git pull</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="gitPushAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/Upload.png</normaloff>../icons/Upload.png</iconset>
   </property>
   <property name="text">
    <string>Отправить</string>
   </property>
   <property name="statusTip">
    <string>git push</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="gitCommitAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/Save.png</normaloff>../icons/Save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="statusTip">
    <string>git commit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="showLogAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Лог приложения</string>
   </property>
  </action>
  <action name="settingsAction">
   <property name="text">
    <string>Настройки...</string>
   </property>
  </action>
  <action name="showBranchesAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ветки</string>
   </property>
  </action>
  <action name="showDomainsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Домены</string>
   </property>
  </action>
  <action name="showProjectsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Проекты</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
